packages/react-components/src/theming/FluentV9ThemeProvider.tsx

```diff
@@ -32,30 +33,26 @@ export const useFluentV9Wrapper = makeStyles({
 });
 
 /**
- * Provider to apply a Fluent V9 theme to react components.
- *
- *
  * @private
  */
 export const FluentV9ThemeProvider = (props: FluentV9ThemeProviderProps): JSX.Element => {
   const { v8Theme, children } = props;
   const v9Theme = createV9Theme(v8Theme);
-
-  // This class wrapper is needed to ensure the useFluentV9Wrapper hook
-  // is called within the context of FluentProvider
-  interface ClassWrapperProps {
-    children: React.ReactNode;
-  }
-  const ClassWrapper = (props: ClassWrapperProps): JSX.Element => {
-    const { children } = props;
-    const fluentV9Wrapper = useFluentV9Wrapper();
-
-    return <div className={fluentV9Wrapper.body}>{children}</div>;
-  };
+  const dir = v8Theme.rtl ? 'rtl' : 'ltr';
 
   return (
-    <FluentProvider theme={v9Theme} dir={v8Theme.rtl ? 'rtl' : 'ltr'}>
-      <ClassWrapper>{children}</ClassWrapper>
-    </FluentProvider>
+    // TextDirectionProvider is needed to fix issue with direction value update in FluentProvider
+    <TextDirectionProvider dir={dir}>
```

Comment:
So this means we can once again apply the provider locally to the chat thread without issue with rtl/ltr?